assets/sass/components/forms.scss

```diff
@@ -409,12 +409,6 @@
         border: none;
     }
 
-    input[type='submit'] {
```

Comment:
Removing this because this was a copy-paste error from the lines above so these styles are already set.